01-Git_Bash_Make/Bash_o_script.script

```diff
@@ -0,0 +1,17 @@
+#! /bin/bash
+
+if [ ! -d ~/tmp ]; then
```

Comment:
/tmp/guesanumber is full path, not relative

Reply:
There is ~